adopt_net0/components/technologies/genericTechnologies/stor.py

```diff
@@ -344,14 +358,21 @@ def init_storage_level(const, t):
             # Cut according to Morales-Espana "LP Formulation for Optimal Investment and
             # Operation of Storage Including Reserves"
             def init_cut_bidirectional(const, t):
-                # output[t]/discharge_rate + input[t]/charge_rate <= storSize
-                return (
-                    self.output[t, self.component_options.main_input_carrier]
-                    / discharge_rate
-                    + self.input[t, self.component_options.main_input_carrier]
-                    / charge_rate
-                    <= b_tec.var_size
-                )
+                if self.flexibility_data["power_energy_ratio"] == "fixedcapacity":
+                    return (
+                        self.output[t, self.component_options.main_input_carrier]
+                        + self.input[t, self.component_options.main_input_carrier]
+                        <= b_tec.var_size
```

Comment:
I am not sure if this is correct (shouldnt be a problem though). I think the RHS should be var_capacity_discharge + var_capacity_charge. In the case that the charging cap and discharging cap are bigger then the size, the current formulation is too tight, in the case they are smaller, it is too loose.